src/main/drivers/dshot.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define DSHOT_MIN_THROTTLE       48
+#define DSHOT_MAX_THROTTLE     2047
+#define DSHOT_3D_FORWARD_MIN_THROTTLE 1048
+
+typedef struct dshotProtocolControl_s {
+    uint16_t value;
+    volatile bool requestTelemetry; // XXX volatile???
+} dshotProtocolControl_t;
+
+void dshotInitEndpoints(float outputLimit, float *outputLow, float *outputHigh, float *disarm, float *deadbandMotor3dHigh, float *deadbandMotor3dLow);
+float dshotConvertFromExternal(uint16_t externalValue);
+uint16_t dshotConvertToExternal(float motorValue);
+
+FAST_CODE uint16_t prepareDshotPacket(dshotProtocolControl_t *const pcb);
```

Comment:
top-level (by value) const is not a part of function signature, it just means `I will not change the value (address) of this pointer, but the underlying value is subject to change`
Doesn't seem like any const is necessary here, as `pcb` is an `out` parameter.

Reply:
Done